app/lottery-external/page.tsx

```
@@ -0,0 +1,206 @@
+import Link from "next/link";
+
+import { FloatingMenu } from "@/app/components/FloatingMenu";
+import {
+  APPLICATION_PERIOD,
+  EXTERNAL_FORM_URL,
+  NOTICES,
+  RESULT_ANNOUNCEMENT,
+  RESULT_PAGE_URL,
+} from "@/app/lottery-external/formConfig";
+import styles from "@/app/lottery-external/lottery-external.module.css";
+import { PartySizeGuide } from "@/app/lottery/[lotteryId]/PartySizeGuide";
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)